spec/bashly/integration/bash_spec.rb

```diff
@@ -8,6 +8,7 @@
       command = "docker run --rm -v $PWD:/app bash:3 bash /app/download"
 
       Dir.chdir "examples/minimal" do
+        system "bashly generate 2>&1 >/dev/null"
```

Comment:
This test was using whichever version of the generated script was lying in the worktree, which was very confusing until I figured out what was happening.

Reply:
But why would the code in the example/minimal folder be any different, and require regeneration?

Reply:
It's not the code in the folder that is different, but rather the way the spec is using it. The `examples_spec.rb` file runs the examples by their `test.sh`, which all have `bashly generate` at the top of them and so regenerate the script before calling it. This spec is different: it calls the generated script directly (it has to, since `bashly` is not available in the `bash:3` container) and so unless we do so explicitly the script is not regenerated in this case.